bip-taro.mediawiki

```diff
@@ -532,17 +530,20 @@ where:
 * <code>prev_asset_witnesses</code>: is a ''nested'' TLV that contains the asset witnesses needed to verify the merging into the target asset leaf
 ** <code>prev_asset_id</code>: references the previous asset input by the input position within the transaction, the asset ID of the previous asset tree, and the asset script hash.
 *** This value is included in any signatures generated within the asset witness itself, serving a purpose similar to the previous outpoint in vanilla Bitcoin.
-*** If this field is all zeroes, then that indicates a ''new'' asset is being created.
+*** If the <code>asset_script_key</code> field is all zeroes, then that indicates a ''new'' asset is being created.
 *** For ''internal'' split verification, this field can be used to save space by including only a single signature/witness that covers all other new splits.
 *** If this type isn't present, then a <code>split_commitment</code> MUST be present.
 ** <code>split_commitment_proof</code>: is used to permit the spending of an asset UTXO created as a result of an asset split. When an asset is split, the non-change UTXO commits to the location of all other splits within an MS-SMT tree. When spending a change UTXO resulting from an asset_split, a normal <code>asset_witness</code> isn't required, instead the owner of the change asset UTXO must prove that it holds a valid split which was authorized by the main transfer transaction.
 *** Outputs with the same <code>split_commitment</code> are said to share a single <code>asset_witness</code> as such outputs are the result of a new asset split. Therefore we only need a single witness and the resulting merkle-sum asset tree to verify a transfer.
-** <code>asset_witness</code>: is a serialized witness in an identical format as Bitcoin's Segwit witness field. This field can only be blank if <code>prev_asset_id</code> is blank.
+** <code>asset_witness</code>: is a serialized witness in an identical format as Bitcoin's Segwit witness field. 
+*** If <code>asset_script_key</code> is all zeroes, the witness will be either
+**** empty, to indicate this asset does not support further issuance. In this case <code>asset_group_key</code> MUST NOT be present.
+**** non-empty, in case the witness will be interpreted according to BIP-341 validtation rules on the <code>asset_group_key<code>. In this case <asset_group_key> MUST be present.
```

Comment:
Link to section in the VM BIP about how to verify it?

Reply:
Done: 7f5b921c9a58e96f7b4ab32958be43927cbeae3e